source/runbooks/set-up-macie.html.md.erb

```diff
@@ -0,0 +1,89 @@
+To do this you should connect to the environment on which you want to create Macie. Note, for accounts with 4 versions (development, test, preproduction, production) you will need to create Macie on each one. In theory you can just set it up in each account and allow one (say production) to use every account to access their data. At the time this document was produced this was not tested but can be used according to an AWS video showing a [demo](https://www.youtube.com/watch?v=8piwEQJJXdo).
+
+AWS also have a brief outline of the purpose of [Macie](https://docs.aws.amazon.com/macie/latest/user/what-is-macie.html) which can be checked to see if this is suitable.
+
+Starting Macie with Terraform is possible and the instructions are shown below. There is a pre-requisite that you need to ensure is completed before this is undertaken. I have also listed the details of setting up jobs. Although they can be set up (another example is shown) it may be beneficial to set these up manually. The example only shows a few of the many items that can be included.
+
+# Terraform code
+
+It is possible to set up Macie using Terraform code. 
+
+To get more detail it is best to do a search for ["aws macie terraform"](https://www.google.com/search?q=aws+macie+terraform&rlz=1C5GCCM_en&oq=&gs_lcrp=EgZjaHJvbWUqCQgAEEUYOxjCAzIJCAAQRRg7GMIDMgkIARBFGDsYwgMyCQgCEEUYOxjCAzIJCAMQRRg7GMIDMgkIBBBFGDsYwgMyCQgFEEUYOxjCAzIJCAYQRRg7GMIDMgkIBxBFGDsYwgPSAQkzMTE4ajBqMTWoAgiwAgE&sourceid=chrome&ie=UTF-8)
+
+Set up a [Macie account](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_account)
+
+Creating a [Macie job](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/macie2_classification_job)
+
+Below there is an example of how the items were set up in Cooker following the instructions listed here.
+
+## Example Terraform code
+
+There is one pre-requisite, for the code shown here, to allow this to work and run successfully. Make sure that the role for **MemberInfrastructureAccess has ace macie2:** set. If this has not been set up correctly (by the Modernisation Platform Team) the terraform listed below will not work. If this is amended in the future, so that it is enabled globally, this comment will be removed. 
+
+The full details of the "finding_publishing_frequency" and "job_type" can be seen in the terraform links listed above.
+
+Full details of this creation, including information on the likes of job type, are shown in the code (macie.tf) in the modernisation-platform-environments repo for cooker. The appropriate environment account reference should be used rather than the "account-code for this environment" shown below.
+
+```
+# Create macie account
+# When logging in as administrator this is not needed but may be for our customers - yet to be fully determined.
+# Account name is MacieAccount. Publishes data every hour (choice are FIFTEEN_MINUTES, ONE_HOUR or SIX_HOURS).
+
+resource "aws_macie2_account" "MacieAccount" {
```

Comment:
I think this whole section of code could just be linked to instead of being written out here if the code exists in example?

Reply:
Not in example at present, see notes above.
If it's needed in example, which I suspect it will be, this can all be amended to replace cooker with example